README.md

```diff
@@ -12,7 +12,7 @@ maven
 <dependency>
     <groupId>com.redhat.darcy</groupId>
     <artifactId>darcy-webdriver</artifactId>
-    <version>0.3.2-SNAPSHOT</version>
+    <version>0.3.3-SNAPSHOT</version>
```

Comment:
I think let's leave this as 0.3.2-SNAPSHOT. When we bump SNAPSHOT versions without a release it is a little odd.

Either that, or I can release 0.3.2 real quick and we merge this to the new 0.3.3-SNAPSHOT development version post release.

What do you think?